src/main/java/org/broadinstitute/hellbender/utils/tsv/SimpleXSVWriter.java

```diff
@@ -200,7 +200,7 @@ public LineBuilder fill(final String filling) {
          * Constructs the line and writes it out to the output
          */
         public void write() {
-            Utils.validate(!Arrays.stream(lineToBuild).anyMatch(Objects::isNull), "Attempted to construct an incomplete line, make sure all columns are filled");
+            Utils.validate(!Arrays.stream(lineToBuild).anyMatch(Objects::isNull), "Attempted to construct an incomplete line, make sure all columns are filled: " + Arrays.toString(lineToBuild));
```

Comment:
Perhaps this String should be lazily constructed (ie., only when the validation check fails) instead of unconditionally on every call to `write()`

Reply:
Made lazy